src/storage/kv/transaction.rs

```diff
@@ -786,7 +783,7 @@ impl Transaction {
         &self,
         key: &VariableSizeKey,
         query_type: QueryType,
-    ) -> Result<(Bytes, u64, u64)> {
+    ) -> Result<(IndexValue, u64, u64)> {
```

Comment:
If this is not returning Bytes, and IndexValue has no method to be accessed publicly, wouldn't this create a problem on surrealdb? Since indexvalue is private, I'd expect the return type to remain the same

Reply:
The actual value is now being resolved.